### PR TITLE
Create LICENSE

### PR DESCRIPTION
**Pull request by hgaiser**

### CODE REVIEW

**Comment by de-vri-es** [comment]
You should also update package.xml, and according to the license we should put a copyright notice on each file. But we can skip that bit imo.

**Comment by de-vri-es** [comment]
Oh, you beat me to it.

**Comment by hgaiser** [comment]
By like a second :p

File: package.xml [hunk 1]
```diff
@@ -3,7 +3,7 @@
 	<name>dr_eigen</name>
 	<version>0.0.0</version>
 	<description>The dr_eigen package</description>
-	<license>all-rights-reserved</license>
+	<license>Apache v2.0</license>
```
Comment (de-vri-es) [inline package.xml]:
```suggestion
	<license>Apache-2.0</license>
```

Should use the SPDX identifier.

**Review by de-vri-es — APPROVED** [review]

**Pull request merged**